connectors/filtering/validation.py

```diff
@@ -255,6 +261,8 @@ def validate(cls, rule):
 
 
 class BasicRuleNoMatchAllRegexValidator(BasicRuleValidator):
+    """Class validating that a basic rule does not use a match all regex."""
```

Comment:
```suggestion
    """BasicRuleNoMatchAllRegexValidator can be used to check that a basic rule does not use a match all regex."""
```

What's the deal with match all regex here?

Reply:
I actually can not recall, what was the exact rationale to prohibit match all regular expressions. Should be somewhere in the original implementation doc.